order/supplier/receive/simple-order-event.md

```diff
@@ -0,0 +1,249 @@
+* `lines`: one or more lines of the order, see [Order lines](#order-lines).
+* `indicators.deliveryOverdue` is true when at least one order line is overdue.
+* `status.processStatus`: is the aggregate of all lines statuses, see [Order process status](#order-process-status).
+* `status.logisticsStatus`: is the aggregate of all lines statuses, see [Order logistics status](#order-logistics-status).
+* `version`: the Tradecloud order version number.
+* `eventDates`: some key order event date/times.
+* `meta`: meta information, including source and trace info, about this messsage.
+* `lastUpdatedAt`: is the latest date time the order has been changed.
+
+### Buyer order
+
+`buyerOrder` contains the buyer order fields:
+
+* `companyId`: the buyer's Tradecloud company identifier.
+* `supplierAccountNumber`: your account number as known in the buyer's ERP system.
+* `description`: a free format additional description of this order added by the buyer.
+* `contact`: the buyer employee responsible for this order.
+* `properties`: are key-value based custom fields, added by the buyer.
+* `notes`: are simple custom fields, added by the buyer.
+* `labels`: value-added services labels on order level.
+* `documents`: contain meta data, objectId or url, of attached documents by the buyer.
+* `orderType`: the order type, one of `Purchase` or `Forecast`.
+
+{% page-ref page="download-document.md" %}
+
+### Supplier order
+
+`supplierOrder` is mostly an echo of your order fields as explained in [Send order response](../send-order-response/)​.
+
+* `buyerAccountNumber`: the buyer account number as known in your ERP system.
+
+{% hint style="warning" %}
+The `buyerAccountNumber` should be set on forehand in the Tradecloud connection with your buyer. You can set the account code when inviting a new connection or in the connection overview in the portal.
+{% endhint %}
+
+### Order status
+
+The order status is the aggregation of all the lines statuses.
+
+#### Order process status
+
+{% hint style="info" %}
+The order **process** status is one of:
+
+* `Issued`: the order is \(re\)issued by the buyer.
+* `InProgress`: the order is under negotiation between buyer and supplier
+* `Confirmed`: the order is completely agreed between buyer and supplier
+* `Rejected`: the order is completely rejected by supplier
+* `Completed`: the order is completed at the buyer
+* `Cancelled`: the order is cancelled by the buyer
+{% endhint %}
+
+#### Order logistics status
+
+{% hint style="info" %}
+The order **logistics** status is one of:
+
+* `Open`: no or partial quantity Produced, ReadyToShip, Shipped or Delivered
+* `Produced`: the order full quantity is produced by the supplier
+* `ReadyToShip`: the order full quantity is ready to be shipped by the supplier
+* `Shipped`: the order full quantity is shipped by the supplier
+* `Delivered`: the order full quantity is delivered at the buyer
+* `Cancelled`: the order is cancelled by the buyer
+{% endhint %}
+
+## Order lines
```

Comment:
This is still part of the `## simpleOrderEvent`
```suggestion
### Order lines
```

Reply:
Headers below may need to be adjusted accordingly.
Or perhaps use: 
- `## simpleOrderEvent header`
- `## simpleOrderEvent lines`

Reply:
Updated, but we did not call it `header` in the API.